errormessage/src/main/kotlin/motif/errormessage/DuplicatedChildParameterSourceHandler.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2018-2019 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package motif.errormessage
+
+import motif.models.DuplicatedChildParameterSource
+
+internal class DuplicatedChildParameterSourceHandler(private val error: DuplicatedChildParameterSource) : ErrorHandler {
+
+    override val name = "DUPLICATED CHILD PARAMETER SOURCE"
+
+    override fun StringBuilder.handle() {
+        appendln("""
+            Child scope contains duplicated child parameter sources:
```

Comment:
Let's make this a little more human-readable: `Multiple child method parameters of the same type:`.